Python/Product/Analysis/MemberResult.cs

```diff
@@ -175,6 +173,34 @@ public struct MemberResult {
             }
             return result.ToString().Trim();
         }
+
+        private static string FormatModuleDocumentation(string doc) {
+            // Module doc does not nave example/signature lines like a function
+            // so just make it flow nicely in the tooltip by removing like breaks.
+            // Preserve double breaks and breaks before the indented text
+            var lines = doc.Replace("\r", string.Empty).Split('\n');
```

Comment:
At least use `StringReader.ReadLine()` here (though I suspect this may be a worthwhile place to optimize to reduce response time of completion lists - then again, resolving documentation lazily would also help significantly there)